cmd/kpod/common.go

```diff
@@ -46,5 +47,10 @@ func getConfig(c *cli.Context) (*libkpod.Config, error) {
 			config.StorageOptions = opts
 		}
 	}
+	if c.Bool("debug") {
+		logrus.SetLevel(logrus.DebugLevel)
+	} else {
+		logrus.SetLevel(logrus.ErrorLevel)
+	}
```

Comment:
Would this be better in an `app.Before`, rather than as a side-effect of `getConfig`?  That way, we'd have debugging support for the things we do before retrieving the configuration in `kpod push`.